pkg/child/child.go

```diff
@@ -187,6 +189,14 @@ func Child(opt Opt) error {
 	if msg.Stage != 1 {
 		return errors.Errorf("expected stage 1, got stage %d", msg.Stage)
 	}
+	// The parent calls child with Pdeathsig, but it is cleared when newuidmap SUID binary is called
+	// https://github.com/rootless-containers/rootlesskit/issues/65#issuecomment-492343646
+	runtime.LockOSThread()
+	err = unix.Prctl(unix.PR_SET_PDEATHSIG, uintptr(unix.SIGKILL), 0, 0, 0)
+	runtime.UnlockOSThread()
```

Comment:
This is incorrect, for PDEATHSIG the parent is the thread, not the process. I'll leave a more detailed comment on the original issue.

Reply:
Could you open an issue?